Quake/host_cmd.c

```diff
@@ -1251,6 +1251,10 @@ static void Host_Loadgame_f (void)
 		entnum++;
 	}
 
+	// Free edicts allocated during map loading but no longer used after restoring saved game state
+	for (int i = entnum; i < sv.num_edicts; i++)
```

Comment:
No C99-ism please: We already have a top-level `i`, so just removing `int ` will be OK.